google-cloud-firestore/test/google/cloud/firestore/client/transaction_test.rb

```diff
@@ -117,6 +119,21 @@
     end
   end
 
+  it "runs a simple query with read only transaction" do
+    expected_query = Google::Cloud::Firestore::V1::StructuredQuery.new(
+      select: Google::Cloud::Firestore::V1::StructuredQuery::Projection.new(
+        fields: [Google::Cloud::Firestore::V1::StructuredQuery::FieldReference.new(field_path: "name")]),
+      from: [Google::Cloud::Firestore::V1::StructuredQuery::CollectionSelector.new(collection_id: "users", all_descendants: false)]
+    )
+    firestore_mock.expect :run_query, query_results_enum, run_query_args(expected_query, new_transaction: read_only_transaction_opt)
+    # firestore_mock.expect :commit, empty_commit_resp, commit_args(transaction: transaction_id)
```

Comment:
Please don't leave commented out code. Either uncomment or delete the line. If the line is meant to be disabled _temporarily_, include a clear comment detailing why it's disabled and when it should be re-enabled.

Reply:
My bad.. Forgot to remove the line. Will fix this.